tests/jballerina-unit-test/src/test/resources/test-src/docerina/fieldLevelDocsProject/src/testModule/test.bal

```diff
@@ -0,0 +1,64 @@
+import ballerina/io;
+
+public const string LKA = "LKA";
+public const string LK = "LK";
+public const string USA = "USA";
+
+public type CountryCode LK|LKA|USA;
+
+# Address record
+#
+# + street - street of the address
+# + city - city of the address
+# + countryCode - country code of the address
+public type Address record {|
+    string street;
+    string city;
+    CountryCode countryCode;
+|};
+
+# Person record
+public type Person record {|
+    # name of the person
+    string name;
+    # age of the person
+    int age;
+    # country code of the person
+    CountryCode countryCode;
+|};
+
+# Apartment record
+#
+# + number - number of the apartment
+# + street - street of the apartment
+# + countryCode - country code of the apartment
+public type Apartment record {|
+    # apartment no
+    int number;
+    # apartment street
+    string street;
+    # apartment country-code
+    CountryCode countryCode;
+|};
+
+# Student object
+#
+# + name - student name
+# + age - student age
+public type Student object {
+    public string name = "Saman";
+    public int age = 15;
+};
+
+# Teacher object
+public type Teacher object {
+    # Teacher name
```

Comment:
Shall we have some test cases with md at field level

Reply:
Added a test case